### ISSUE TITLE
PDF passes validation with type 3 fonts that aren't embedded

### ISSUE THREAD

**Issue by Corvwyn** [veraPDF/veraPDF-library#1458]
I'm converting some PDF files to PDF/A-1b. After the conversion we use veraPDF to verify that these are valid PDF/A-1b.

In one instance we have a PDF file that veraPDF validates as valid PDF/A-1b. This file contains two fonts called T3Font_0 and T3Font_1, that aren't embedded. 

Is this the correct behaviour, or is there something I'm missing? Is there something special about type 3 fonts that doesn't require them to be embedded? 

I can provide an example pdf if needed, I just need to ask if it's ok to share first.

**Comment by THausherr** [comment]
Type 3 fonts are not really fonts, this is a collection of PDF content streams, one per glyph, so them not being embedded might be a misunderstanding.

**Comment by Corvwyn** [comment]
@THausherr Thanks for the info. In that case, it makes sense that veraPDF validates the PDF this way. 

Adobe Acrobat lists them as type 3 fonts that aren't embedded.

The main problem is that the library we use to concatenate these files see them as unembedded fonts. I guess we might have to create a support ticket, so they load type 3 fonts in a different way.

Thanks for the quick reply!

**Comment by bdoubrov** [comment]
@Corvwyn feel free to upload the files to this issue. I would  double check if indeed these fonts are Type3 ones. Font names might be misleading sometimes.

**Comment by Corvwyn** [comment]
@THausherr Great. I will upload the pdf soon, I just need to ask if it's ok.

**Comment by THausherr** [comment]
Yeah there are type 3 fonts on page 60. And it's like I described.

Btw I found a different problem. VeraPDF claims it is a PDF/A file, PDFBox Preflight claims it isn't. The reason is that the file has `/SMask (None)` but "None" as a string instead of as a name. One of us is wrong 😂

**Comment by Corvwyn** [comment]
Hmm. What a predicament 😛

**Comment by petervwyatt** [comment]
I will also highlight [PDF Errata 118](https://github.com/pdf-association/pdf-issues/issues/118) and [PDF Errata 6](https://github.com/pdf-association/pdf-issues/issues/6) - words such as "absent" or "present" in all PDF specs are ambiguous and very likely not what is desired. This may be why...

**Comment by bdoubrov** [comment]
We've implemented additional object type checks, so that `/SMask (None)` is reported as a validation error. This fix is available in the latest dev builds.

**Comment by MaximPlusov** [comment]
Included into release 1.28
